libafl/src/executors/command.rs

```diff
@@ -283,6 +287,20 @@ where
     }
 }
 
+impl<OT, S, T> HasTimeout for CommandExecutor<OT, S, T>
+where
+    S: State,
+    T: CommandConfigurator<S::Input>,
```

Comment:
don't use S::Input.

Reply:
@tokatoka not really sure what I should use instead?

Reply:
why do you need S: State?

Reply:
If you want to get the input type, then just S: HasCorpus. then the corpus type is <S::Corpus as Corpus>::Input